perception/sandbox/eyehandcal/readme.md

```diff
@@ -77,7 +83,8 @@ Run `collect_data_and_cal.py` to run the calibration process by sampling from th
 
 ```bash
 $ collect_data_and_cal.py --help
-usage: collect_data_and_cal.py [-h] [--ip IP] [--datafile DATAFILE] [--overwrite] [--marker-id MARKER_ID] [--calibration-file CALIBRATION_FILE] [--points-file POINTS_FILE] [--num-points NUM_POINTS] [--time-to-go TIME_TO_GO] [--imagedir IMAGEDIR]
+usage: collect_data_and_cal.py [-h] [--seed SEED] [--ip IP] [--datafile DATAFILE] [--overwrite] [--marker-id MARKER_ID] [--calibration-file CALIBRATION_FILE] [--points-file POINTS_FILE] [--num-points NUM_POINTS]
```

Comment:
--proj-func arguments could have more intuitive names like wristCam and steadCam.